### DISCUSSION TITLE
Dynamic import not working with turbo repo internal package

### DISCUSSION THREAD

**Discussion by jgimitola** [vercel/turborepo#4718] [Help] [answered]
### Summary

Hi, I am having some issues with a project i am developing. I am using turborepo and Nextjs. In the monorepo i have an `app` for the Nextjs app and a `package` where the ui components are implemented.

![image](https://user-images.githubusercontent.com/47477274/233517125-d4c31320-cf08-4bc3-894c-3049bf0be42f.png)

The thing is that when i try to use dynamic with my internal package nothing happens, but if i create a component in the app project and use dynamic the new chunk is created.

![image](https://user-images.githubusercontent.com/47477274/233516873-09e48146-2a77-4bcd-b3ae-90d4e1c66cb9.png)

Third request from top to bottom shows the new chunk for Field (local component) but there is no new chunk for OrangeCtaButton (package component)

I don't know what I could be doing wrong. Any idea what could be happening?

I have posted this discussion on next repo but i have not had an answer yet. I don't know if this is problem is a bug or something I am doing wrong.

**Comment by anthonyshew** [comment, marked as answer]
Your ask in the Next.js repo is likely the correct spot. If there is a bug to be had here, Turborepo is incredibly unlikely to be the cause.
